.idea/runConfigurations/_template__of_Gradle.xml

```diff
@@ -16,7 +16,6 @@
     <ExternalSystemDebugServerProcess>true</ExternalSystemDebugServerProcess>
     <ExternalSystemReattachDebugProcess>true</ExternalSystemReattachDebugProcess>
     <DebugAllEnabled>false</DebugAllEnabled>
-    <ForceTestExec>false</ForceTestExec>
```

Comment:
we shouldn't add this file as part of this PR, even if we decide to commit it, see discussion on Slack.

Reply:
Thanks!